src/tracer/tracer.test.js

```diff
@@ -109,6 +110,36 @@ describe('tracer', () => {
     });
   });
 
+  test('startTrace - timeout timer - send execution tags', (done) => {
+    const timeout = 2000;
+    const testBuffer = 50;
+
+    new EnvironmentBuilder().awsEnvironment().applyEnv();
+    const token = 't_1234';
+    globals.TracerGlobals.setTracerInputs({ token });
+
+    const { event, context } = new HandlerInputsBuilder().withTimeout(timeout).build();
+    TracerGlobals.setHandlerInputs({ event, context });
+    const functionSpan = getFunctionSpan(event, context);
+    tracer.startTrace(functionSpan).then(() => {
+      const span1 = { id: 'SomeRandomHttpSpan' };
+      SpansContainer.addSpan(span1);
+      globals.ExecutionTags.addTag('key1', 'value1');
+
+      setTimeout(() => {
+        const requests = AxiosMocker.getRequests();
+        //1 for start span, 1 for SomeRandomHttpSpan + 1 enrichmentSpan
```

Comment:
the comment is a bit misleading. I'm not sure that I understood it myself...
Maybe change it to: "2 requests: 1 for the start span, and another for the timeout spans" 🤔